modules/seqexec/server/src/main/scala/seqexec/server/nifs/NifsControllerEpics.scala

```diff
@@ -0,0 +1,174 @@
+// Copyright (c) 2016-2018 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package seqexec.server.nifs
+
+import cats.data.OptionT
+import cats.effect.IO
+import cats.effect.Timer
+import cats.implicits._
+import edu.gemini.spModel.gemini.nifs.NIFSParams.{ ReadMode => LegacyReadMode }
+import edu.gemini.spModel.gemini.nifs.NIFSParams.{ EngReadMode => LegacyEngReadMode }
+import edu.gemini.seqexec.server.nifs.DhsConnected
+import org.log4s.getLogger
+import scala.concurrent.ExecutionContext
+import seqexec.model.dhs.ImageFileId
+import seqexec.server.ObserveCommand
+import seqexec.server.Progress
+import seqexec.server.ProgressUtil
+import seqexec.server.SeqexecFailure
+import seqexec.server.failUnlessM
+import shapeless.tag
+import squants.Seconds
+import squants.Time
+import squants.time.TimeConversions._
+
+object NifsLookupTables {
+
+  val readModeLUT: Map[LegacyReadMode, Int] = Map(
+    LegacyReadMode.BRIGHT_OBJECT_SPEC -> 1,
+    LegacyReadMode.MEDIUM_OBJECT_SPEC -> 1,
+    LegacyReadMode.FAINT_OBJECT_SPEC -> 1
```

Comment:
Why do they all map to 1? Because they are legacy?

Reply:
Yeah, isn't that silly but that's what the original TCL code does. I prefer to keep it though we could just default to 1